.changeset/calm-ears-provide.md

```diff
@@ -0,0 +1,18 @@
+---
+"@premieroctet/next-admin": minor
+---
+
+📄 add option for custom pages
+
+In the `options`, add
+
+```tsx
+pages: {
+  "/custom": {
+    title: "Custom page",
+    component: CustomPage,
```

Comment:
We should add an option to hide the page, for example if want to combine it with custom actions.